storybook/src/docs/app-icon.docs.mdx

```diff
@@ -0,0 +1,135 @@
+```
+
+### Apple Touch Icon
+
+<PngIconAppleTouch />
+
+```html
+<link rel="apple-touch-icon" href="apple-touch-icon.png" />
+```
+
+### Web App Icons
+
+<PngIconsWebApp />
+
+Include `icons` object in `*.webmanifest` (see [Usage](#usage)):
+
+```json
+{
+  // ...
+  "icons": [
+    { "src": "/icon-192.png", "type": "image/png", "sizes": "192x192" },
+    { "src": "/icon-512.png", "type": "image/png", "sizes": "512x512" }
+  ]
+  // ...
+}
+```
+
+Link the Web Manifest in the `<head>`:
+
+```html
+<link rel="manifest" href="app.webmanifest" />
+```
+
+## Usage
+
+First, install these assets as they come with the [Amsterdam Design System Assets](https://www.npmjs.com/package/@amsterdam/design-system-assets):
+
+```sh
+npm i @amsterdam/design-system-assets
+```
+
+Then you can either use the post install script or manually copy and link the assets.
+
+### (A) Automatic script
+
+Use the [`.postinstall.sh`](https://github.com/Amsterdam/design-system/tree/develop/proprietary/assets/.postinstall.sh) script from the root of your new **Vite React**, **Next.js**, legacy React or generic HTML project to automatically link the assets:
+
+```sh
+# Make sure you are in the root directory of your project!
+sh node_modules/@amsterdam/design-system-assets/.postinstall.sh
+```
+
+> If an `index.html` is found in the root or in `public` the script will also insert the links in the HTML's `<head>` tag. This will not be the case for default Next.js applications.
+
+> This script can work for any project that has an `index.html` file in the root or `public` directory.
+
+### (B) Manual configuration
+
+You can manually [symlink (symbolic link)](https://en.wikipedia.org/wiki/Symbolic_link#POSIX_and_Unix-like_operating_systems) and copy the files to the root or a `public` folder next to where the `index.html` is located in your project. An advantage of symlinking is that it tracks changes to the assets when there are packages updates.
+
+Symlinking [the entire favicon directory](https://github.com/Amsterdam/design-system/tree/develop/proprietary/assets/favicon) and copying the [Web Manifest](https://github.com/Amsterdam/design-system/tree/develop/proprietary/assets/app.manifest) can be done as follows:
+
+```sh
+# `cd` to the directory where you want to make the symlinks.
+ln -s node_modules/@amsterdam/design-system-assets/favicon appicons
+cp node_modules/@amsterdam/design-system-assets/manifest/app.webmanifest app.webmanifest
+```
+
+> Note: for the symlink the last argument in above command cannot be a path, but can only be the file or directory name. I.e. you need to be in the directory where you want to make the symbolic link.
+
+> Note that undoing a symlink can be done by simply removing the created file or directory link:
+>
+> ```sh
+> rm -r appicons app.webmanifest
+> ```
+
+Make sure that the Web Manifest (`*.webmanifest`), stating the app's name and referencing the PNG icons, is also available in the same location:
```

Comment:
Maybe note that some servers need to be configured to serve a .webmanifest file as JSON MIME type: https://www.w3.org/TR/appmanifest/#using-a-link-element-to-link-to-a-manifest

Reply:
I have some experience with PWA's as well, but that was a while ago. I should read into how browsers handle PWA stuff these days.

Reply:
Still the same for the pwa stuff, a ".webmanifest " is something that was added since I worked on it. Appears to be the same as a manifest.json file, just the mime type needs to work on the server.

Reply:
I added a note with that link.